mm2src/adex_cli/src/adex_config.rs

```diff
@@ -151,7 +154,12 @@ impl AdexConfigImpl {
         let adex_path_str = cfg_path
             .to_str()
             .ok_or_else(|| error_anyhow!("Failed to get cfg_path as str"))?;
-        rewrite_json_file(self, adex_path_str)
+        rewrite_json_file(self, adex_path_str)?;
+        #[cfg(unix)]
+        {
+            set_file_permissions(adex_path_str, CFG_FILE_PERM_MODE)?;
```

Comment:
is this advised? Wouldn't it be better/safer to assume config file has correct permission setting as opposed to implementing a de-facto "chmod" into mm2?
Rly concerned over likely unneeded filestream ops and speaking general - against this.

Reply:
I advised him to this as it's creating the config JSON that is used by a separate process, mm2. 

There could be a better solution, but without this, the seed will be readable by any user on the system.

Reply:
Yep, [it was adviced](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1871#discussion_r1259568425). 

The command: `adex-cli config set -u http://localhost:77873 -p` creates the configuration file and setting permissions could look quite essential.

cc: @Alrighttt

Reply:
It's not totally necessary. Maybe a warning while creating this configuration would suffice since the target audience of this app is presumably power users or at least users familiar with a terminal.

Reply:
[done](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1932/commits/5e7f9a1721b84137472bea5b80e498714e15c293)